### PR TITLE
Init Command [minor]

### PR DESCRIPTION
**Pull request by ric-evans**
closes https://github.com/Observation-Management-Service/ewms-pilot/issues/51

### CODE REVIEW

**Comment by ric-evans** [comment]
I'm adding tests

File: ewms_pilot/pilot.py [hunk 1]
```diff
@@ -101,7 +105,21 @@ async def consume_and_reply(
...
     try:
+        # Init command
+        if init_cmd:
+            # TODO - call func that makes subprocess shell using asyncio-tasks w/ housekeeper.basic_housekeeping()
+            await run_init_command(
+                init_cmd,
+                init_timeout,
+                housekeeper,
+                staging_dir,
+            )
```
Comment (dsschult) [inline ewms_pilot/pilot.py:120]: Would it be better to run the init cmd before opening the queues?  That way we don't need to do "housekeeping" and if there is a node error we don't even connect to the queues.

Comment (ric-evans) [reply ewms_pilot/pilot.py:120]: That's a good idea. As for housekeeping, I separated queue-related housekeeping from "basic" housekeeping, although there's not much going on in "basic" housekeeping currently

Comment (ric-evans) [reply ewms_pilot/pilot.py:120]: We could put htchirp stuff in basic housekeeping

Comment (dsschult) [reply ewms_pilot/pilot.py:102]: That's an interesting idea

**Review by dsschult — APPROVED** [review]
Approved pending the tests passing

**Pull request merged**